src/operator_repo/checks/__init__.py

```diff
@@ -79,8 +79,14 @@ class Fail(CheckResult):
     kind = "failure"
 
 
-SUPPORTED_TYPES = [("operator", Operator), ("bundle", Bundle)]
-Check = Callable[[Union[Repo, Operator, Bundle]], Iterable[CheckResult]]
+SUPPORTED_TYPES = [
+    ("operator", Operator),
+    ("fbc", OperatorCatalogList),
```

Comment:
What about calling it "catalog" instead of fbc? It sounds more understandable to me.

Reply:
Changed to catalog to be more clear.